app/src/service/ocr/index.ts

```diff
@@ -0,0 +1,33 @@
+import { createWorker } from "tesseract.js";
+
+import { createLogger } from "../../utils/logger";
+import * as parsers from "./parser";
+
+type OcrOptions = {
+  debug: boolean;
+};
+
+// main logger
+export const logger = createLogger("ocr-parser", {
+  enabled: process.env.LOG_LEVEL === "debug",
```

Comment:
Another opinionated addition that I don't have strong feelings about. The OCR parser will print debugging information when the LOG_LEVEL is set to debug.